ci-operator/step-registry/hypershift/conformance/hypershift-conformance-chain.yaml

```diff
@@ -153,8 +154,12 @@ chain:
       if [[ -n "${TEST_SKIPS}" ]]; then
           TESTS="$(openshift-tests run --dry-run --provider "${TEST_PROVIDER}" "${TEST_SUITE}")"
           echo "${TESTS}" | grep -v "${TEST_SKIPS}" >/tmp/tests
+          if [[ "${CLUSTER_TYPE}" == "kubevirt" ]]; then
+            echo "${TESTS}" | grep "\[sig-kubevirt\]" >>/tmp/tests
+            KUBVIRT_NO_SKIP="| grep -v '\[sig-kubevirt\]'"
+          fi
           echo "Skipping tests:"
-          echo "${TESTS}" | grep "${TEST_SKIPS}" || { exit_code=$?; echo 'Error: no tests were found matching the TEST_SKIPS regex:'; echo "$TEST_SKIPS"; return $exit_code; }
+          echo "${TESTS}" | grep "${TEST_SKIPS}" ${KUBEVIRT_NO_SKIP:-} || { exit_code=$?; echo 'Error: no tests were found matching the TEST_SKIPS regex:'; echo "$TEST_SKIPS"; return $exit_code; }
```

Comment:
i'm unsure about this approach. It appears this short circuits the ability to skip a `sig-kubevirt` test using the `TEST_SKIPS` env var.

Reply:
good catch, thanks.
I've modified the script to take that into account as well, so `TEST_SKIPS` will be honored if running on kubevirt platform.
in a nutshell, the logic is as follows:

1. get the skipped tests list excluding the trailing `[openshift/conformance/parallel]` (if exists), and store in an env var.
2. strip the trailing `\|` from that env var.
3. remove backslashed from that env var.
4. from ALL tests, extract the `[sig-kubevirt]` ones, and filter out the kubevirt skipped tests, using perl regex with grep). non-kubevirt skips were handled earlier.
5. if no `TEST_SKIPS` has been provided, we filter out nothing and add all `[sig-kubevirt]` test cases to the minimal suite.